src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs

```diff
@@ -152,14 +152,14 @@ public static WallType Wrap(Autodesk.Revit.DB.WallType ele, bool isRevitOwned)
 
         public static View3D Wrap(Autodesk.Revit.DB.View3D view, bool isRevitOwned)
         {
-            if (view.IsPerspective)
+            if (!view.IsTemplate)
```

Comment:
Exclude template views for now, waiting for API guild's response whether we should totally ignore them or there are other work around.